modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -204,7 +204,9 @@ const Modal = ({
 					>
 						{url ? (
 							<>
-								{loading && <ClayLoadingIndicator />}
+								{loading && (
+									<ClayLoadingIndicator className="position-absolute" />
+								)}
```

Comment:
Thanks for the feedback @markocikos I didn't realize it! I change the css to have one isolated version with the cadmin-selector and one without it. Since modals can be configured to be isolated or not I think this is needed. Let me know what do you think :)

Reply:
Nice! :+1: